internal/depinject/container.go

```diff
@@ -52,9 +52,13 @@ func (c *Container) build() error {
 	}
 
 	for _, node := range c.registry.Nodes() {
-		for _, dep := range node.Dependencies() {
+		for i, dep := range node.Dependencies() {
 			source, err := c.registry.Get(dep)
-			if err != nil {
+			// If the last dependency is not found, and the node is variadic,
+			// we continue without error.
+			if err != nil && node.IsVariadic() && i == len(node.Dependencies())-1 {
+				continue
```

Comment:
nit: would probably be clearer if you use `break` here and line 93, but don't have very strong feelings about this

Reply:
i feel like the break keyword has a connotation of exiting the loop "early" which feels wrong to me since semantically we're just trying to skip even tho the condition requires it to be the last element so functionally equivalent